pyproject.toml

```diff
@@ -91,15 +91,15 @@ requires = [
     max-complexity = 40
 
   [tool.ruff.pylint]
-    max-args = 16
+    max-args = 20
     max-branches = 32
     max-returns = 16
     max-statements = 124
 
   [tool.ruff.per-file-ignores]
     "*/__init__.py" = ["F401", "F403"] # Allow unused imports and star imports
     "kornia/__init__.py" = ["I001"] # Allow unsorted imports
-    "test/*" = ["S101", "S311", "BLE"] # allow assert, random, ignore BLE
+    "test/*" = ["S101", "S311", "BLE", "RUF012"] # allow assert, random, ignore BLE, mutable class attr
```

Comment:
`RUF012` is worthwhile to fix in this PR or a followup PR...
% `ruff rule RUF012`
# mutable-class-default (RUF012)

Derived from the **Ruff-specific rules** linter.

## What it does
Checks for mutable default values in class attributes.

## Why is this bad?
Mutable default values share state across all instances of the class,
while not being obvious. This can lead to bugs when the attributes are
changed in one instance, as those changes will unexpectedly affect all
other instances.

When mutable value are intended, they should be annotated with
`typing.ClassVar`.

## Examples
```python
class A:
    mutable_default: list[int] = []
```

Use instead:
```python
from typing import ClassVar


class A:
    mutable_default: ClassVar[list[int]] = []
```